### DISCUSSION TITLE
/code-driven-approach-theme-vscode-webview/

### DISCUSSION THREAD

**Discussion by comment-migrator** [estruyf/eliostruyf-comments#590] [Comments]
A code-driven approach to theme your VS Code webview
        
[https://www.eliostruyf.com/code-driven-approach-theme-vscode-webview/](https://www.eliostruyf.com/code-driven-approach-theme-vscode-webview/)

**Comment by chartsNgraphs** [comment]
Although using the variables in .css works for me, I see that document.html no longer contains computed styles anymore. on document.html, out of the hundreds of styles, no --vscode styles exist when accessed from javascript. 

Did something change perhaps?

**Reply by estruyf** [reply]
I just validated it, and it is still working as before. The variables are set on the `html` element.

<img width="1752" alt="image" src="https://github.com/user-attachments/assets/7244c192-9230-4276-938c-fdf2dc1bec99" />
